web/components/ConnectionCard.tsx

```diff
@@ -11,7 +11,7 @@ interface IProps {
 
 export function ConnectionCard(props: IProps) {
     return <div className={style.card} onClick={props.onClick}>
-        <img src={props.imageSrc} />
+        <img alt="" src={props.imageSrc} />
```

Comment:
```suggestion
        <img alt="Service logo" src={props.imageSrc} />
```

Reply:
It has the serviceName printed right next to it.
And a user of a screen reader is unlikely to be interested in the presence of a logo.